styles.css

```diff
@@ -67,3 +66,241 @@ ul {
   font-size: 20px;
   margin-top: 25px;
 }
+
+/* .........About myself section........... */
+.About-myself {
+  width: 90%;
+  margin: auto;
+}
+
+.Recent-work {
+  margin-top: 80px;
+}
+
+.Recent {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #1c1a19;
+  height: 83px;
+}
+
+.line {
+  border: 1px solid #212121;
+}
+
+.about-img {
+  height: 237px;
+  margin-top: 50px;
+}
+
+.Multi-Post-Heading {
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+}
+
+.Multi-Post-stories {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  color: #091e42;
+  margin-top: 12px;
+  line-height: 24px;
+}
+
+.Multi-Post-ul {
+  width: 100%;
+  margin-bottom: 15px;
+}
+
+.Multi-Post-li {
+  border: 1px solid #8993a4;
+  padding: 9px 12px;
+  margin-right: 10px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+}
+
+.Multi-Post-project {
+  align-items: center;
+  gap: 10px;
+  width: 124px;
+  height: 48px;
+  background: #ff6b00;
+  color: #fff;
+  border-style: none;
+}
+
+/* ...........work section.......... */
+.work-container {
+  width: 90%;
+  margin: auto;
+  background-color: #fff;
+  padding-bottom: 120px;
+}
```

Comment:
As requested in the project requirements: 
![image](https://user-images.githubusercontent.com/42799579/198420726-76f7621c-3743-4f2c-adc7-e3338045a9d9.png) 
Please make sure to use both a grid and flexboxes to organize your projects section 😃. We need to make sure that you are able to work with both of these tools :wink:.